### ISSUE TITLE
Test Issue for testing the Haack SentimentBot

### ISSUE THREAD

**Issue by haacked** [haacked-demos/azure-sentiment-analysis#1]
If you're going to comment on an issue in this repository just to test the bot out, please use this one. Also note that I'm using the trial for the [Text Analytics API service](https://azure.microsoft.com/en-us/services/cognitive-services/text-analytics/) so I might run out and this might stop working.

If you don't see the comment update, try refreshing the page. Or perhaps what you wrote isn't negative/positive enough.

I know you're testing out a sentiment bot, but you can still be negative and civil to each other. Please abide by the [code of conduct](https://github.com/haacked-demos/azure-sentiment-analysis/blob/master/CODE_OF_CONDUCT.md).

**Comment by haacked** [comment]
Well that's just the stupidest thing I've ever heard! This is idiotic and foolish.

_Sentiment Bot Says: Hey now, let's keep it positive. (Score: {sentimentScore})_

**Comment by haacked** [comment]
Hey, great job in fixing the bug! You forgot to make it a format string so it says `sentimentScore` but anyone could have made that mistake. You really did wonderful to figure that out and fix it. Kudos to you, you smart and kind fellow.

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.886396944522858)_

**Comment by haacked** [comment]
This comment is pretty neutral. I don't expect the bot to say anything here.

**Comment by theunrepentantgeek** [comment]
Fantastic idea. Love it. #testing

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.994409441947937)_

**Comment by bartelink** [comment]
I reckon you've made it too easy with all your hard work. Now everyone is going to start writing those easy ML bots and we'll be overrun. But I do welcome our new sentiment analysis overlord 🙇.

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.983574330806732)_

EDIT: Sentibot lost my edits when I fixed 2 typos; oh well, I'm still going to hide the evidence by editing and seeing if it can deal with that.

**Comment by rossdargan** [comment]
Yeah... Nice job Einstein. Just what we need, robots taking our jobs. They might be smart, but I wonder how well they deal with sarcasm. I'd love to compliment you for this neat ideas, but I have a sentiment score to game.

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.989894032478333)_

**Comment by rossdargan** [comment]
I do realise that was more a test of the cognitive services than your bot! A great example of serverless architecture. For me this is the future containerisation won't deliver on. Just wish there was an open standard for it!

Anyway nice demo!

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.856989204883575)_

**Comment by XVincentX** [comment]
Test: I'm so sad about this project

_Sentiment Bot Says: Hey now, let's keep it positive. (Score: 0.023887425661087)_

**Comment by joelwkall** [comment]
Does it deal with negations? This bot is not absolutely terrible. The blog post you wrote about it  couldn't be calles worthless, by any stretch of the imagination. It doesn't at all fill me with disgust, quite the opposite, really.

**Comment by tiesmaster** [comment]
God damn!!! This is the stupidest idea I've ever seen. F*ck off with this sh*t!!!

_Sentiment Bot Says: Hey now, let's keep it positive. (Score: 0.13743531703949)_

**Comment by XVincentX** [comment]
@joelwkall It seems like you made it crash.

_Sentiment Bot Says: Hey now, let's keep it positive. (Score: 0.131100386381149)_

**Comment by tiesmaster** [comment]
Alright, this is actually really awesome!! The bot responded really quick. Nice job! (And kudos to @joelwkall for making it crash :P)

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.943233251571655)_

**Comment by joelwkall** [comment]
@XVincentX I think it just interpreted my comment as neutral. Which makes sense; "not terrible" is not a very good sentiment, but it's not terrible either. 

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.878625273704529)_

**Comment by haacked** [comment]
Hmmm, maybe I should always post the sentiment score. Just to see how it deals with neutral comments on the edge.

**Comment by lee-dohm** [comment]
Yeah, I think always posting the sentiment score would be best for a demo :+1:

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.873439610004425)_

**Comment by lee-dohm** [comment]
What about the sentiment of emoji? ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ 

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 1)_

**Comment by lee-dohm** [comment]
Or emoticons? <3 <3 <3 <3 <3 <3 <3 

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.999939680099487)_

**Comment by lee-dohm** [comment]
To be honest, I didn't think it would catch them :laughing:

**Comment by Giorgio-Montagnoli** [comment]
I think it's a great idea, but, honestly, this is so language dependent that I cannot use it and I'm kinda sad. 

_Sentiment Bot Says: Hey now, let's keep it positive. (Score: 0.184679120779037)_

**Comment by kushalbhabra** [comment]
Not sure this was genius or brilliant.

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.962514996528625)_

**Comment by KenBonny** [comment]
The language service is weird. Being sad gives you a lower score than cursing.

_Sentiment Bot Says: Hey now, let's keep it positive. (Score: 0.00512534379959106)_

**Comment by KenBonny** [comment]
@KenBonny great going, nitwit.

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.91748183965683)_

**Comment by mehalick** [comment]
Love it! :joy_cat:

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.945624649524689)_

**Comment by kushalbhabra** [comment]
How about we aggregate the sentiments, or showing trending sentiment for a issue as a whole!?

**Comment by pedrolamas** [comment]
This is quite cool!

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.995809555053711)_

**Comment by yasmoradi** [comment]
:-S

**Comment by dchalom** [comment]
This is the best geek project I have not ever seen. I couldn't fail to disagree with you less.

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.94588440656662)_

**Comment by brianbolger** [comment]
Oh, this is terribly good

**Comment by haacked** [comment]
@Giorgio-Montagnoli the language service API supports multiple languages. For example, it can do language detection. For this demo, I hard-coded it to English. But it wouldn't be too hard to make it work for more languages.

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.835910081863403)_

**Comment by haacked** [comment]
I've made a change so it should update neutral comments with a score.

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.947903275489807)_

**Comment by haacked** [comment]
Oh, this is terribly good.

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.902262330055237)_

**Comment by haacked** [comment]
Hmmm

_Sentiment Bot Says: How very netural. (Score: 0.740837693214417)_

**Comment by haacked** [comment]
There we go.

_Sentiment Bot Says: How very netural. (Score: 0.256691217422485)_

**Comment by dchalom** [comment]
I would like to say that this is good.
I would like to say.

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.920957803726196)_

**Comment by Giorgio-Montagnoli** [comment]
@Haacked Greeeeat! Now I absolutely have to test it in Italian! Thanks for your feedback!

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.886873304843903)_

**Comment by dchalom** [comment]
This is wicked cool!

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.880703449249268)_

**Comment by haacked** [comment]
@Giorgio-Montagnoli there's a page where you can try out the text analysis API directly. https://azure.microsoft.com/en-us/services/cognitive-services/text-analytics/

_Sentiment Bot Says: How very netural. (Score: 0.707798361778259)_

**Comment by sergiorgiraldo** [comment]
I could make much better than this. Go study

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.901820182800293)_

**Comment by sergiorgiraldo** [comment]
This is fantastic piece of code. Kudos!

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.840919137001038)_

**Comment by sergiorgiraldo** [comment]
Meh

_Sentiment Bot Says: Hey now, let's keep it positive. (Score: 0.0499474108219147)_

**Comment by aaronpowell** [comment]
If I don't survive, tell my wife "Hello"

_Sentiment Bot Says: How very netural. (Score: 0.745844423770905)_

**Comment by djturner1984** [comment]
I strongly disagree with the idea that this is not a good bot

_Sentiment Bot Says: Hey now, let's keep it positive. (Score: 0.0446923971176147)_

**Comment by yaakov-h** [comment]
I am so excited to destroy the world! 😈 MUAHUAHAHAHA

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.998911023139954)_

**Comment by mikeyb137** [comment]
Mae hyn yn wych. Mae trafod yr iaith Gymraeg yn wych. Yn ogystal â'r cyfieithydd a ddefnyddiwyd i ysgrifennu'r sylw hwn hyd yn oed.

_Sentiment Bot Says: How very netural. (Score: 0.732595920562744)_

**Comment by mikeyb137** [comment]
Mae hyn yn sbwriel. Mae popeth amdano yn arogli drwg. Rwy'n casáu popeth ohono. Dyma'r peth gwaethaf erioed.

_Sentiment Bot Says: How very netural. (Score: 0.753245174884796)_

**Comment by DaveInKentucky** [comment]
I really dislike not writing code.

_Sentiment Bot Says: How very netural. (Score: 0.203382879495621)_

**Comment by DaveInKentucky** [comment]
I hate not writing code.

_Sentiment Bot Says: How very netural. (Score: 0.203382879495621)_

**Comment by DaveInKentucky** [comment]
I'm surprised at how stupendously neutral this is.

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.909034550189972)_

**Comment by fidelisrafael** [comment]
Estou postando em português, boa ideia! Achei genial!

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.865597665309906)_

**Comment by fidelisrafael** [comment]
Merda, caraleo, bosta, lixo, cuzão!

**Comment by fidelisrafael** [comment]
This shit only works in english?

_Sentiment Bot Says: Hey now, let's keep it positive. (Score: 0.0240471065044403)_

**Comment by fidelisrafael** [comment]
Looks like this awesome tool only works in English! Amazing!

_Sentiment Bot Says: How very netural. (Score: 0.243664294481277)_

**Comment by fidelisrafael** [comment]
I'm trying to be cool! Love it! :heart: 

_Sentiment Bot Says: Thanks for keeping it so positive! (Score: 0.963072597980499)_

**Comment by petrkotas** [comment]
Hey is this work really working? I do not see it. I do not believe it is grading negative texts. It does not work.

**Comment by wayne-o** [comment]
This sucks ballz

**Comment by aussiedev** [comment]
This is awesome!

**Comment by taicoding** [comment]
Me parece muy interesante, deberia reconocer otros idiomas tambien, eso seria espectaular.

**Comment by anuragde** [comment]
This is a cool idea to gather statistics on a repo.

**Comment by prankshaw** [comment]
Changed the whole structure to build dependencies from different source but at least it is the latest version, so will be helpful in security of application with all the extra hours I have to put to make it successful
